publons/PublonsHandler.inc.php

```diff
@@ -104,7 +104,7 @@ function exportReview($args, $request) {
                 $reviewId = $reviewAssignment->getId();
                 $reviewFormResponseDao =& DAORegistry::getDAO('ReviewFormResponseDAO');
                 $reviewFormElementDao =& DAORegistry::getDAO('ReviewFormElementDAO');
-                $reviewFormElements = $reviewFormElementDao->getReviewFormElements($reviewFormId);
+                $reviewFormElements = $reviewFormElementDao->getByReviewFormId($reviewFormId)->toArray();
```

Comment:
This is caused by the definition change of `ReviewFormElementDAO` class, in ojs3, `getReviewFormElements` function is replaced by `getByReviewFormId` which will return a `DAOResultFactory`